tests/lib/cli.js

```diff
@@ -357,6 +388,21 @@ describe("cli", function() {
         });
     });
 
+    describe("when given patterns to ignore", function() {
+        it("should not process any matching files", function() {
+            var ignorePaths = ["a", "b"];
+
+            verifyCLIEngineOpts(
+                ignorePaths.map(function(ignorePath) {
+                    return "--ignore-pattern " + ignorePath;
```

Comment:
Can you move this out to a local variable? It will make it easier to see what you're doing here.